docs/pages/versions/unversioned/guides/setup-native-firebase.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Setup Native Firebase
+---
+
+Because the native Firebase SDK requires build-time configuration, you cannot use it with the managed Expo client from the App Store. Luckily there are a few workarounds that we'll cover in this guide.
+
+- **Bare-Workflow (Universal):** You can add any native packages you want, but you cannot currently build a bare-workflow project in the cloud with Expo's build service.
```

Comment:
not necessary to define bare workflow here again, we can just link to https://docs.expo.io/versions/v36.0.0/introduction/managed-vs-bare/#bare-workflow